server/api/crud/files.py

```diff
@@ -35,6 +38,23 @@ def get_filestat(
     ):
         return self._get_filestat(schema, path, user, auth_info, secrets=secrets)
 
+    def delete_files_with_project_secrets(
```

Comment:
```suggestion
    def delete_artifact_data()
```

Reply:
The method is in files.py, so it can be a genetic function that is not unique to artifact data

Reply:
not sure I understand. the "how" is given on the function argument. why do u need it on function name?